packages/db-utils/src/index.ts

```diff
@@ -0,0 +1,7 @@
+export const getDb = () => {
```

Comment:
What do you think about renaming the function to `getDbOrNull`? 

```suggestion
export const getDbOrNull = () => {
```

Then the use would be clear that the function could return `null` and eliminate the need for a redundant comment (currently this function could return null)

```
let Db = getDbOrNull();
if (Db) {...}
```

Reply:
I don't really like `getDbOrNull`, I think it would confuse me if I wasn't familiar. Maybe the comments I wrote are unnecessary.